sceptre/scipool/config/bmgfki/sc-product-s3-private-enc.yaml

```diff
@@ -40,7 +40,11 @@ sceptre_user_data:
       Info:
         LoadTemplateFromURL:  'https://{{stack_group_config.admincentral_cf_bucket}}.s3.amazonaws.com/{{stack_group_config.service_catalog_library}}/v1.1.15/s3/sc-s3-encrypted-ra.yaml'
       Name: 'v1.1.15'
-    - Description:  'Retain bucket on SC product termination. {{ range(1, 10000) | random }}'
+    - Description:  'Retain bucket on SC product termination.'
       Info:
         LoadTemplateFromURL:  'https://{{stack_group_config.admincentral_cf_bucket}}.s3.amazonaws.com/{{stack_group_config.service_catalog_library}}/v1.1.23/s3/sc-s3-encrypted-ra.yaml'
       Name: 'v1.1.23'
+    - Description:  'Enable intelligent tiering. {{ range(1, 10000) | random }}'
```

Comment:
Are we sure we want to make this available for all the accounts right away? Wouldn't it be wiser to only make it available to develop first and after a test run enable it on the other accounts?

Reply:
nah, that's not fun.  let's spice it up and throw it out there.